jenkins/scripts/fetch_logs.sh

```diff
@@ -20,12 +20,19 @@ source "${CI_DIR}/utils.sh"
 
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-# Run feature tests, e2e tests, main and release* tests in the Frankfurt region
-if [[ "${TESTS_FOR}" == "feature_tests"* ]] || [[ "${TESTS_FOR}" == "e2e_tests"* ]] ||
-        [[ "${UPDATED_BRANCH}" == "main" ]] || [[ "${UPDATED_BRANCH}" == "release"* ]]; then
+# Run:
+#   - e2e features, clusterctl-upgrade tests in the Frankfurt region
+#   - ansible, e2e, basic integration, k8s-upgrade tests in Karlskrona region
+#   - keep tests in dev2 project Karlskrona region
+if [[ "${KEEP_TEST_ENV}" == true ]]; then
```

Comment:
```suggestion
if [[ "${KEEP_TEST_ENV}" == "true" ]]; then
```
For all of those in this PR.

Reply:
done